### PR TITLE
Add Nequix MP PFT

### PR DESCRIPTION
**Pull request by teddykoker**
Adds the phonon-fine-tuned Nequix MP PFT model, trained on the PBE MDR phonon database. The [preprint](https://arxiv.org/abs/2601.07742) reports more than 30% lower thermal-conductivity error while RMSD and F1 degrade by less than 2%.

| Model | Params | κ_SRME |
| --- | ---: | ---: |
| Nequix MP | 707K | 0.446 |
| **Nequix MP PFT** | 707K | **0.306** |

| Model | RMSD | F1 |
| --- | ---: | ---: |
| Nequix MP | 0.0853 | 0.751 |
| **Nequix MP PFT** | 0.0866 | 0.747 |

Fine-tuning uses only MP IDs present in MPtrj, although the phonon calculations add DFT data beyond the original Materials Project set. Direct training on finite-displacement data did not improve performance, while direct analytical-Hessian supervision did. The process is reproducible from the [upstream instructions](https://github.com/atomicarchitects/nequix/?tab=readme-ov-file#phonon-fine-tuning-pft), but no fine-tuning script is included here.

- Adds the MDR Phonon dataset and training split to `datasets.yml`.
- Adds the `EFSH_G` target for energy with gradient-based forces, stress, and Hessian plus an energy-parity visualization.
- Adds dataset-level compliance flags and makes model compliance checks use that metadata.

### CODE REVIEW

File: models/nequix/nequix-mp-1-pft.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  pyyaml: 6.0.2
+  matscipy: 1.1.1
+  h5py: 3.14.0
+  wandb-osh: 1.2.2
+  cloudpickle: 3.1.1
+
+metrics:
+  phonons:
+    kappa_103:
+      pred_file: models/nequix/nequix-mp-1-pft/2026-01-12-kappa-103-FIRE-dist=0.03-fmax=1e-4-symprec=1e-5.json.gz
```
Comment (CompRhys) [inline models/nequix/nequix-mp-1-pft.yml:127]: file has a different name on figshare, can you run the eval scripts (`scripts/evals/kappa.py`) to populate the kappa metrics?

File: models/nequix/nequix-mp-1-pft.yml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      pred_file_url: https://figshare.com/files/60965506
+  geo_opt:
+    pred_file: models/nequix/nequix-mp-1-pft/2026-01-12-wbm-IS2RE-FIRE.jsonl.gz
```
Comment (CompRhys) [inline models/nequix/nequix-mp-1-pft.yml:132]: file has a different name on figshare, can you run the eval scripts (`scripts/evals/geo_opt.py`) to populate the geo metrics?

File: models/nequix/nequix-mp-1-pft.yml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    pred_file_url: https://figshare.com/files/60965515
+    struct_col: nequix_structure
+  discovery:
+    pred_file: models/nequix/nequix-mp-1-pft/2026-01-12-wbm-IS2RE.csv.gz
```
Comment (CompRhys) [inline models/nequix/nequix-mp-1-pft.yml:154]: file has a different name on figshare, can you run the eval scripts (`scripts/evals/discovery.py`) to populate the discovery metrics?

File: data/datasets.yml [hunk 4]
```diff
@@ -614,3 +614,53 @@ MatPES r2SCAN:
   description: |
     A dataset containing 387,897 structures from ~300K molecular dynamics simulations,
     providing potential energy surfaces that are valuable for training ML interatomic potentials.
+
+MDR Phonon PBE:
```
Comment (CompRhys) [inline data/datasets.yml:654]: Because it provides a big uplift for kappa and is small in scope and somewhat in the spirit to the compliant data set I am not opposed to saying that this could be considered "compliant in spirit" because even though it does introduce new data that amount of data is upper bounded in scope. In principle the definition of compliant in the paper would let people use the PBEsol data from MP. It's better to use consistent DFT settings to more clearly understand the effects.

> We define the [MP v.2022.10.28](https://docs.materialsproject.org/changes/database-versions#v2022.10.28) database release as the maximum allowed training set for any compliant model submission. Models may train on the complete set of relaxation frames, or any subset thereof such as the final relaxed structures. Any subsets of the energies, forces and stresses are valid training targets. In addition, any auxiliary tasks such as predicting electron densities, magnetic moments, site-partitioned charges and so on that can be extracted from the output of the DFT calculations are allowed for multi-task learning.

I pushed for keeping the compliant concept because I think is useful for architectural and procedural changes like PFT. Obviously some things at MPtrj scale won't scale but it is cheaper to iterate and has thus far had more models using the same training data for consistency which also aids comparison. 

my two cents but will leave decision to @janosh to think about when he looks at this :)

**Review by CompRhys — COMMENTED** [review]
Looks mostly good, can you pull the recent changes and run the two plot generation scripts I added to the checklist?

The principle thing to fix is the naming of the figshare files and the generation of the metrics for the yml.

**Comment by teddykoker** [comment]
> > Thanks @CompRhys! I updated the filenames on figshare to match, and ran the plots and kappa evals. I am having issues running the other eval infrastructure due to http 403 errors on figshare. Do you know if there is a workaround for this? Hopefully it should work on your end with the filenames fixed.
> 
> I was also seeing this, if you individually click all the links and download and drag to the right folder it works. For me I already had most files downloaded so that didn't take much time for me to go around it manually for the other models I merged in this burst of activity. Sorry to not be much help... I think figshare must have changed their robots policy or something.

Just uploaded the metrics! Let me know if there is anything else you need.

**Review by janosh — APPROVED** [review]
LGTM! thanks for the submission @teddykoker 

phonon fine-tuning is a very welcome addition! i've been bullish on that approach for 2+ years, even before we fully finished the discovery part of the benchmark
